common/app/views/support/HtmlCleaner.scala

```diff
@@ -500,6 +500,16 @@ case class DropCaps(isFeature: Boolean) extends HtmlCleaner {
         case _ =>
       }
     }
+
+    document.getElementsByTag("h2").foreach{ h2 =>
+        if (h2.text() == "* * *") {
```

Comment:
OK, I'd seen it within a `<p>` but you're right it seems to be within an `<h2>` on recent pieces.

:+1: 

cc @na7hangood :crying_cat_face: